src/Package/Impl/Commands/RHistory/VsRHistoryCommandFactory.cs

```diff
@@ -38,14 +40,16 @@ internal class VsRHistoryCommandFactory : ICommandFactory {
             var interactiveWorkflow = _interactiveWorkflowProvider.GetOrCreate();
             var sendToReplCommand = new SendHistoryToReplCommand(textView, _historyProvider, interactiveWorkflow);
             var sendToSourceCommand = new SendHistoryToSourceCommand(textView, _historyProvider, interactiveWorkflow, _contentTypeRegistry, _textViewTracker);
-            var appShell = VsAppShell.Current;
+            var shell = VsAppShell.Current;
```

Comment:
Actually, there is no command that requires `ICoreShell` and doesn't require `interactiveWorkflow`, so I think you can just remove this variable completely.